app/assets/javascripts/components/revisions/diff_viewer.jsx

```diff
@@ -89,7 +98,15 @@ const DiffViewer = React.createClass({
   },
 
   findParentOfFirstRevision() {
-    const wikiUrl = `https://${this.props.first_revision.wiki.language}.${this.props.first_revision.wiki.project}.org`;
+    const wikiProject = this.props.revision.wiki.project;
+    let wikiUrl;
+
+    if (wikiProject === 'wikidata') {
+      wikiUrl = `https://${wikiProject}.org`;
+    } else {
+      wikiUrl = `https://${this.props.first_revision.wiki.language}.${wikiProject}.org`;
+    }
+
```

Comment:
This could all be just:

```javascript
const wikiUrl = this.wikiUrl();
```

The wiki for `first_revision` will always be the same as the wiki for `revision`, so no need to duplicate any of that logic.